_posts/ja/2021-06-16-cardcoins-rbf-batching-ja.md

```diff
@@ -0,0 +1,17 @@
+---
+title: 'Field Report: Using RBF and Additive Batching'
+permalink: /ja/cardcoins-rbf-batching/
+name: 2021-06-16-cardcoins-rbf-batching-ja
+slug: 2021-06-16-cardcoins-rbf-batching-ja
+type: posts
+layout: post
+lang: ja
+version: 1
+excerpt: >
+   "Additive batching" is a scheme whereby additional outputs are
```

Comment:
@azuchi can you translate this as appropriate?

Reply:
@bitschmidty Thanks, I translated (cd85de7).